gizmo.py

```diff
@@ -0,0 +1,56 @@
+class Gizmo:
+    def __init__(self, name):
+        self.name = name
+
+    def speak(self):
+        print (self.name)
+
+def hello(name, country = "Finland"):
```

Comment:
Little style thing: usually we don't put spaces here between the parameter and its default value.
```suggestion
def hello(name, country="Finland"):
```